tests/catboost/test_catboost_model_export.py

```diff
@@ -56,13 +56,30 @@ def read_yaml(path):
     params=[
         cb.CatBoost(MODEL_PARAMS),
         cb.CatBoostClassifier(**MODEL_PARAMS),
+        cb.CatBoostRanker(**MODEL_PARAMS),
         cb.CatBoostRegressor(**MODEL_PARAMS),
     ],
-    ids=["CatBoost", "CatBoostClassifier", "CatBoostRegressor"],
+    ids=["CatBoost", "CatBoostClassifier", "CatBoostRanker", "CatBoostRegressor"],
 )
 def cb_model(request):
     model = request.param
     X, y = get_iris()
+
+    if isinstance(model, CatBoostRanker):
+        # the ranking task requires setting a group_id
+        # we are creating a dummy group_id here that doesn't make any sense for the Iris dataset, but is ok for testing
+        # if the code is running correctly
+
+        indices = np.arange(len(candidates))
```

Comment:
@danielgafni Thanks for the PR! Is `candidates` defined somewhere? Looks like it's not.